include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -215,6 +216,12 @@ using __buffer_allocator =
     sycl::buffer_allocator;
 #endif
 
+#if __LIBSYCL_VERSION >= 50700
+#    define SYCL_CL cl
+#else
+#    define SYCL_CL
+#endif
+
```

Comment:
The result of my investigations - now the next version of SYCL after this change is 5.7.0

Reply:
About inline namespace cl
It was introduced at PR https://github.com/intel/llvm/pull/974 :
    Commit 67e4655b (open)
    by Alexey Bader, 26.12.2019 09:16:44
    committed by Romanov Vlad
    parent b6806eaf

Reply:
So my assumption - we may simple exclude cl namespace from our code.

Reply:
But also we have the second part of this question: should we replace includes like
     #include <CL/sycl.hpp>
to
     #include <sycl.hpp>

Right now it will work correctly, but in LLVM this change was made later then inline namespace cl:
    Signed Commit ee93fbc9
    by aelovikov-intel, 13.07.2022 22:25:50
    committed by GitHub
    parent 261290b4

[SYCL] Move SYCL headers from CL/ to sycl/ (#6407)

So my proposal - don't change includes now.